src/components/TodoComponents/TodoForm.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const TodoForm = props => {
+    return (
+        <form onSubmit={props.addTask}>
+            <input 
+                type="text"
+                onChange={props.handleChange}
+                value={props.value}
```

Comment:
```suggestion
                value={props.value}
```
Does this line of code actually work?